boms/bom/pom.xml

```diff
@@ -1,294 +1,304 @@
-        <version>5.2</version>
-      </dependency>
+	<dependencyManagement>
+		<dependencies>
+			<!-- *** INTERNAL DEPENDENCIES *** -->
+			<dependency>
+				<groupId>com.devonfw.java.boms</groupId>
+				<artifactId>devon4j-minimal-bom</artifactId>
+				<version>${devon4j.version}</version>
+				<type>pom</type>
+				<scope>import</scope>
+			</dependency>
+			<!-- *** EXTERNAL DEPENDENCIES *** -->
+			<!-- Library with general utilities as well as I18N and exception support -->
+			<dependency>
+				<groupId>net.sf.m-m-m</groupId>
+				<artifactId>mmm-util-nls</artifactId>
+				<version>${mmm.util.version}</version>
+			</dependency>
+			<dependency>
+				<groupId>net.sf.m-m-m</groupId>
+				<artifactId>mmm-util-exception</artifactId>
+				<version>${mmm.util.version}</version>
+			</dependency>
+			<dependency>
+				<groupId>net.sf.m-m-m</groupId>
+				<artifactId>mmm-util-validation</artifactId>
+				<version>${mmm.util.version}</version>
+			</dependency>
+			<!-- JSR 250 for component/bean lifecycle management -->
+			<dependency>
+				<groupId>javax.annotation</groupId>
+				<artifactId>jsr250-api</artifactId>
+				<version>1.0</version>
+			</dependency>
+			<!-- JSR 330 for dependency injection -->
+			<dependency>
+				<groupId>javax.inject</groupId>
+				<artifactId>javax.inject</artifactId>
+				<version>1</version>
+			</dependency>
+			<!-- JSR 250 for common annotations (component-bean lifecycle, security, 
+				etc.) -->
+			<dependency>
+				<groupId>javax.annotation</groupId>
+				<artifactId>javax.annotation-api</artifactId>
+				<version>1.2</version>
+			</dependency>
+			<!-- Library with general utilities -->
+			<dependency>
+				<groupId>com.google.guava</groupId>
+				<artifactId>guava</artifactId>
+				<version>${guava.version}</version>
+			</dependency>
+			<!-- Library with advanced collection support -->
+			<dependency>
+				<groupId>org.apache.commons</groupId>
+				<artifactId>commons-collections4</artifactId>
+				<version>4.1</version>
+			</dependency>
+			<!-- Support for money datatype including operations/calculations -->
+			<dependency>
+				<groupId>org.javamoney</groupId>
+				<artifactId>moneta</artifactId>
+				<version>1.3</version>
+				<type>pom</type>
+			</dependency>
+			<dependency>
+				<groupId>cglib</groupId>
+				<artifactId>cglib</artifactId>
+				<version>3.2.5</version>
+			</dependency>
+			<!-- DataBase Connection Pooling -->
+			<dependency>
+				<groupId>org.apache.commons</groupId>
+				<artifactId>commons-dbcp2</artifactId>
+				<version>2.0.1</version>
+			</dependency>
+			<!-- Logging API -->
+			<dependency>
+				<groupId>org.slf4j</groupId>
+				<artifactId>slf4j-api</artifactId>
+				<version>1.7.28</version>
+			</dependency>
+			<!--Spring cloud dependencies BOM -->
+			<dependency>
+				<groupId>org.springframework.cloud</groupId>
+				<artifactId>spring-cloud-dependencies</artifactId>
+				<version>Greenwich.RELEASE</version>
```

Comment:
Could you please move the version number to a property in main/devon4j/pom.xml next to the spring boot version property and add a comment, that the spring cloud and spring boot versions have to match.

Reply:
ok i will thank you